src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -357,6 +359,13 @@ Copyright (c) .NET Foundation. All rights reserved.
                         Crossgen2RuntimeIdentifiers="@(Crossgen2SupportedRids, '%3B')"
                         />
 
+    <KnownILCompilerPack Include="Microsoft.DotNet.ILCompiler"
+                         TargetFramework="net7.0"
+                         ILCompilerPackNamePattern="Microsoft.NETCore.App.Crossgen2.**RID**"
```

Comment:
ILCompilerPackNamePattern should be `runtime.**RID**.Microsoft.DotNet.ILCompiler`